src/main/java/Duke.java

```diff
@@ -1,10 +1,271 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+
+
```

Comment:
remember to separate the files and provide javadoc when you reach the appropriate stages 😄